setup/_pattoo/docker.py

```diff
@@ -174,6 +173,7 @@ def install(container_name, config_files, verbose=True):
 
     # Build pattoo container
     image_check('pattoo')
+    print('Building Pattoo Container. This may take up to 10 minutes...')
```

Comment:
Say "_This could take some time..._" as it's more generic. We cannot guarantee the response time of another person's computer and network.